--- genders-1.22/src/extensions/java/Makefile.am.orig	2012-08-28 01:25:09.000000000 +0200
+++ genders-1.22/src/extensions/java/Makefile.am	2017-12-03 15:42:01.733745725 +0100
@@ -19,7 +19,7 @@
 noinst_HEADERS  = src/Gendersjni.h
 lib_LTLIBRARIES = src/libGendersjni.la
 
-src_libGendersjni_la_CXXFLAGS = -D_REENTRANT \
+src_libGendersjni_la_CFLAGS = -D_REENTRANT \
 			    -I $(srcdir)/../../libgenders/
 
 src_libGendersjni_la_SOURCES = src/Gendersjni.c
--- genders-1.22/src/extensions/python/Makefile.am.orig	2011-01-07 00:11:33.000000000 +0100
+++ genders-1.22/src/extensions/python/Makefile.am	2017-12-03 16:36:51.170374833 +0100
@@ -17,13 +17,13 @@
 	$(PYTHON) genderssetup.py build
 
 install:
-	$(PYTHON) genderssetup.py install --prefix=$(PYTHON_DESTDIR)/$(prefix) --exec-prefix=$(PYTHON_DESTDIR)/$(exec_prefix)
+	$(PYTHON) genderssetup.py install --prefix=$(PYTHON_DESTDIR)/$(prefix) --exec-prefix=$(PYTHON_DESTDIR)/$(exec_prefix) --optimize=2
 
 pure_install:
-	$(PYTHON) genderssetup.py install --prefix=$(PYTHON_DESTDIR)/$(prefix) --exec-prefix=$(PYTHON_DESTDIR)/$(exec_prefix)
+	$(PYTHON) genderssetup.py install --prefix=$(PYTHON_DESTDIR)/$(prefix) --exec-prefix=$(PYTHON_DESTDIR)/$(exec_prefix) --optimize=2
 
 install-data-local:
-	$(PYTHON) genderssetup.py install --prefix=$(PYTHON_DESTDIR)/$(prefix) --exec-prefix=$(PYTHON_DESTDIR)/$(exec_prefix)
+	$(PYTHON) genderssetup.py install --prefix=$(PYTHON_DESTDIR)/$(prefix) --exec-prefix=$(PYTHON_DESTDIR)/$(exec_prefix) --optimize=2
 
 clean: 
 	rm -rf build
